addons/cookoff/functions/fnc_cookOff.sqf

```diff
@@ -113,22 +113,35 @@ if (local _vehicle) then {
 
         [_vehicle, _fnc_FlameEffect, 12] call _fnc_FlameEffect; // recursive function
 
+        private _randomPosition = [getPos _vehicle, 100, random 360] call BIS_fnc_relPos;
```

Comment:
Ammo starting to cookoff and crew sometime gets killed. 100m is a pretty safe distance.